homework_stack.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+class Stack {
+    private $elements;
+
+    public function __construct()
+    {
+        $this->elements = array(); 
+    }
+
+    public function push($ele)
+    {
+        $this->elements[] = $ele;
+    }
+    
+    public function pop()
+    {
+        if (!$this->isEmpty()) { 
+            array_pop($this->elements[sizeof($this->elements) - 1]); // unset use to destroy a variable (https://www.php.net/manual/en/function.unset)
```

Comment:
Are you trying to get the last element of the stack? So, it's unnecessary to add `sizeof($this->elements) - 1` indexing.

Reply:
It cause an error

![image](https://user-images.githubusercontent.com/83938068/117937472-3ab9f680-b330-11eb-8329-af4ad485da68.png)

Reply:
As per your feedback, the sentence in the `pop` method isn't correct.
I should change the argument of `array_pop` from an incorrect state into just specifying an input array.